src/main/java/seedu/address/model/person/Insurance.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.model.person;
+
+/**
+ * Represents a Person's Insurance in the address book.
+ * Guarantees: immutable; is always valid
+ */
+public abstract class Insurance {
```

Comment:
Good use of inheritance here for the Insurance class, makes it easily extendible in the future 👍